### ISSUE TITLE
Feature Request: Menu separator and sub-menu

### ISSUE THREAD

**Issue by TofPlay** [sveinbjornt/Platypus#89]
Hi,
Do we have an instruction like `MENUSEPARATOR` to specify a menu separator (red in the screenshot):
![image](https://cloud.githubusercontent.com/assets/1082222/21921775/e9d3bcae-d96a-11e6-8e9f-4af2416f563b.png)

Example:
```bash
echo "MENUITEMSPERATOR|<main menu>|<sub menu1>|..."
```
And another instruction like `SUBMENU` and `SUBMENUITEM` to define sub-menu  items of sub-menu (red in the screenshot):
![image](https://cloud.githubusercontent.com/assets/1082222/21922015/4742b308-d96c-11e6-8bb1-60a1ecaae598.png)

Example:
```bash
echo "SUBMENU|Sub-menu on root"
echo "SUBMENU|<main menu>|<sub menu1>|...|My Sub-menu under submenu"
echo "SUBMENUITEM|<main menu>|<sub menu1>|...|My Sub-menu under submenu|Option on my sub-menu"
```

In these examples `<main menu>`, `<sub menu1>` and others are optionals. They are only used to access on a sub-menu

**Comment by sveinbjornt** [comment]
An output line of "----\n" in Status Menu mode creates a menu separator item.

The submenu idea is interesting, although I don't think that would be the syntax I would go for.  Will do for 5.2.

**Comment by sveinbjornt** [comment]
Working in master. Will be released as part of 5.2. Different syntax and non-recursive, though, e.g.:

    SUBMENU|Title|Item1|Item2|Item3
